backend/src/ee/services/group/group-dal.ts

```diff
@@ -6,7 +6,14 @@ import { DatabaseError } from "@app/lib/errors";
 import { buildFindFilter, ormify, selectAllTableCols, TFindFilter, TFindOpt } from "@app/lib/knex";
 import { OrderByDirection } from "@app/lib/types";
```

Comment:
Ignore comment placement

Application testing: It seems endpoint like project list has been broken.